Resources/Textures/Clothing/Uniforms/Jumpskirt/color.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039 and modified by Flareguy for Space Station 14",
+  "copyright": "Taken from vgstation at commit https://github.com/vgstation-coders/vgstation13/commit/993de554ecfc30cc45f8340819c5a60ef1eee7e1 and modified for SS14 by Flareguy & P3MOIRA. Jumpskirt variants made using sprites tgstation at commit https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039 and modified by Flareguy for SS14",
```

Comment:
Is it correct that the jumpskirt variants are using the tg commit but the jumpsuits are from vg..? It's fine if it is but it looks like something you missed.

Reply:
The jumpsuits only use sprites from vgstation, but the jumpskirt uses specifically the skirt part from the tg commit. It's pretty awkward wording, but I'm not sure how else to phrase it - suggestions would be welcome.

Reply:
"Taken from vgstation at commit https://github.com/vgstation-coders/vgstation13/commit/993de554ecfc30cc45f8340819c5a60ef1eee7e1 and modified by Flareguy & P3MOIRA, with the lower half of the jumpskirt taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039 and modified by Flareguy for SS14."